test/Serilog.Settings.Configuration.Tests/DllScanningAssemblyFinderTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.IO;
+
+using Xunit;
+
+using Serilog.Settings.Configuration.Assemblies;
+
+namespace Serilog.Settings.Configuration.Tests
+{
+    public class DllScanningAssemblyFinderTests : IDisposable
+    {
+        readonly string _privateBinPath;
+
+        public DllScanningAssemblyFinderTests()
+        {
+            var d1 = GetOrCreateDirectory("bin1");
```

Comment:
should `"bin1"`, `"bin2"` and `"bin3"` be moved to constants instead of using *magic strings* multiple time ?

Reply:
Okay